### PR TITLE
Add lecture material on basic SQL queries

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

**Comment by ethanwhite** [comment]
Aggregation material is in process, but I thought I'd get the first chunk in for review.

File: materials/sql-basic-queries.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+---
+layout: page
+element: notes
+title: Basic Queries
+language: SQL
+---
+
+> Remember to
+>
+> *  download [`portal_mammals.sqlite`](https://ndownloader.figshare.com/files/2292171).
```
Comment (brymz) [inline materials/sql-basic-queries.md]: extra space?

File: materials/sql-basic-queries.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+* We can also use functions.
+
+```
+SELECT species_id, ROUND(hindfoot_length/1000.0, 2)
+FROM surveys;
+```
+
+
+> Do Exercise 2 - [SELECT]({{ site.baseurl }}/exercises/Basic-queries-select-SQL.md)
```
Comment (brymz) [inline materials/sql-basic-queries.md]: Minor edit: Do [Exercise 2 - SELECT]({{ site.baseurl }}/exercises/Basic-queries-select-SQL.md).

File: materials/sql-basic-queries.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+
+### Filtering
+
+* Use `WHERE` to select only the rows meeting certain criteria.
+
+```
+SELECT hindfoot_length
+FROM surveys
+WHERE species_id = 'DS'
```
Comment (brymz) [inline materials/sql-basic-queries.md]: add `;`

File: materials/sql-basic-queries.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+```
+
+```
+SELECT species_id
+FROM surveys
+WHERE hindfoot_length >= 30
```
Comment (brymz) [inline materials/sql-basic-queries.md]: add `;`

File: materials/sql-basic-queries.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+```
+SELECT year, month, day, species_id, hindfoot_length
+FROM surveys
+WHERE species_id = 'DS' AND year > 1990 AND hindfoot_length IS NOT NULL;
+```
+
+* If you want only NULL values, use `IS NULL` instead
+
+> Do Exercise 3 - [WHERE]({{ site.baseurl }}/exercises/Basic-queries-where-SQL.md)
```
Comment (brymz) [inline materials/sql-basic-queries.md]: Same minor edit: Do [Exercise 3 - WHERE]({{ site.baseurl }}/exercises/Basic-queries-where-SQL.md).

File: materials/sql-basic-queries.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+* Use a list to sort by multiple columns.
+
+```
+SELECT genus, species
+FROM species
+ORDER BY taxa, genus, species;
+```
+
+> Do Exercise 4 - [ORDER BY]({{ site.baseurl }}/exercises/Basic-queries-order-by-SQL.md)
```
Comment (brymz) [inline materials/sql-basic-queries.md]: ... Do [Exercise 4 - ORDER BY]({{ site.baseurl }}/exercises/Basic-queries-order-by-SQL.md).

**Comment by brymz** [comment]
Looks great! Just some real minor changes for consistency. We can get this is and I can make the edits if you prefer.

**Comment by ethanwhite** [comment]
> We can get this is and I can make the edits if you prefer.

If you don't mind that would be great. Thanks!

**Pull request merged**